editor/src/components/canvas/ui-jsx-canvas-renderer/scene-component.tsx

```diff
@@ -54,7 +55,11 @@ function childUnchanged(prevChild: ReactChild, nextChild: ReactChild): boolean {
   if (typeof prevChild === 'string' || typeof prevChild === 'number') {
     return nextChild === prevChild
   } else if (React.isValidElement(prevChild)) {
-    return React.isValidElement(nextChild) && prevChild.type === nextChild.type
+    return (
+      React.isValidElement(nextChild) &&
+      prevChild.type === nextChild.type &&
+      shallowEqual(prevChild.props, nextChild.props)
```

Comment:
actually this should probably be `deepEqual` because props.style (if present) will blow shallow equality

Reply:
Fixed in 1b203a4e.